doc/quickstart/osm_quickstart.rst

```diff
@@ -106,6 +96,84 @@ PostgreSQL for map data, but does not work directly with OSM files.
 * Homepage: https://wiki.openstreetmap.org/wiki/Osm2pgsql
 * Tutorial: https://learnosm.org/en/osm-data/osm2pgsql/
 
+
+Working with OSM data in QGIS
+================================
+
+OSGeoLive comes with OSM Data for the city where the global `FOSS4G conference <https://foss4g.org>`_ takes place.
+You find the data in the file system at /usr/local/share/data/osm/ or in the PostgreSQL database osm_local.
+
+Start QGIS and have a look at the data.
+
+#. Open the `Desktop GIS` folder.
+#. Double click on the `QGIS` icon.
+
```

Comment:
Maybe add how to load the data from the database
#. Go the browser panel on the left of the screen.
#. Navigate to PostGIS and then to the database osm_local. You will find the data in the schema public.
#. Drag the layers from the browser panel and drop them in the map frame.